packages/mc-scripts/src/utils/update-application-id-in-custom-application-config.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs');
+const rcfile = require('rcfile');
+const prettier = require('prettier');
+const babel = require('@babel/core');
+const {
+  getCustomApplicationConfigPath,
+} = require('@commercetools-frontend/application-config');
+
+function updateApplicationIdInCustomApplicationConfig(applicationId) {
```

Comment:
I was wondering if we could maybe just update the applicationId using a regular expression to make this process simpler.

Maybe something like:
```
configFileContents.replace(/("env":.*"production":.*"applicationId":\s?)"(\w*)"/, '$1"[REPLACER]"')
```

(_perhaps this is more complex than I can see_)